newa/cli.py

```diff
@@ -300,8 +301,12 @@ def cmd_jira(ctx: CLIContext, issue_config: str) -> None:
 
 
 @main.command(name='schedule')
+@click.option(
+    '--arch',
+    default='',
```

Comment:
`None` would be better, making `arch` optional - it's easier to detect for linters than an empty string. One can easily pass an empty string where non-empty string is expected, but linters will stop us from passing `None` to functions expecting a string.

Reply:
Fixed.